### PR TITLE
docs(security-policy): call to action and more info

### PR DESCRIPTION
**Pull request by lirantal**
- Adding a call to action top link that is more inviting to quickly report a vulnerability
- Cosmetic spaces

FYI I recently saw this custom `SECURITY.MD` https://github.com/unshiftio/url-parse/commit/d7b582ec1243e8024e60ac0b62d2569c939ef5de

### CODE REVIEW

File: processes/responsible_disclosure_template.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 This project participates in the Responsible Disclosure Policy program for the Node.js Security Ecosystem.
 
+To report a Security Vulnerability [click here](https://hackerone.com/nodejs-ecosystem)
```
Comment (Trott) [inline processes/responsible_disclosure_template.md]: How about this?:

```markdown
To report a security vulnerability, [use hackerone](https://hackerone.com/nodejs-ecosystem).
```

Comment (lirantal) [reply processes/responsible_disclosure_template.md]: I'm not a native English speaker but I was looking for something more inviting that calls for action. For me when I read "use hackerone" it gives me the feeling that I'm redirected elsewhere and it's someone else's responsibility.

Maybe we can try different phrases like one of those:


- [ ] Vulnerabilities can be reported directly to us via [HackerOne](https://hackerone.com/nodejs-ecosystem)
- [ ] 🚔Security Vulnerabilities can be reported directly to us via [HackerOne](https://hackerone.com/nodejs-ecosystem)
- [ ] Report a security vulnerabilities to us through the [HackerOne platform](https://hackerone.com/nodejs-ecosystem)

which in your opinion sounds better or maybe suggest a different wording?

Comment (Trott) [reply processes/responsible_disclosure_template.md]: Any of your three suggestions work for me. Another idea: Make it a single bullet-point call-to-action:

```markdown
* [Report a security vulnerability](https://hackerone.com/nodejs-ecosystem)
```

...which would render something like this:

* [Report a security vulnerability](https://hackerone.com/nodejs-ecosystem)

Anyway, all of those work for me, so whatever you and/or others prefer!

Comment (lirantal) [reply processes/responsible_disclosure_template.md]: Thanks Rich, that suggestion sounds good! I'll go ahead and update for it.

**Pull request merged**